src/internal/operations/backup.go

```diff
@@ -589,6 +596,11 @@ func (op *BackupOperation) persistResults(
 			opStats.writeErr)
 	}
 
+	if opStats.gc == nil {
+		op.Status = Failed
+		return errors.New("data population never completed")
```

Comment:
do we want to fail here? other option is to set `op.Status = NoData` and return nil. Not saying I know the right answer, just raising the option

Reply:
we do want to fail.  But I"m glad you pointed it out, because the other stats needed to get moved above this check.